thunder/tests/opinfos.py

```diff
@@ -2547,6 +2548,7 @@ def div_sample_generator(op, device, dtype, requires_grad, **kwargs):
             pytest.mark.xfail,
             "test_core_vs_torch_consistency",
             executors=("nvfuser",),
+            dtypes=(datatypes.bool8, datatypes.bfloat16, datatypes.float16, datatypes.float32),
```

Comment:
Thanks for adding this line. We should probably follow-up on enabling these tests! It's worrying that nvfuser and torch div appear to be inconsistent (as best our tests can tell). Maybe we have a singularity issue?

fyi @beverlylytle -- this might be interesting for you to take a look at

Reply:
I took a look at these tests, and I don't see a singularity issue.  Instead I see two unrelated issues.  

First, Thunder produces a float64 result from float32 input whereas torch maintains the type.  I don't understand where the conversion is happening.  

Second, one of the sample inputs has a float (ie, not a tensor) as the denominator.  The float16 version of this fails with "Tensor-likes are not close".  If the denominator is replaced by a tensor with empty shape and dtype float16, it passes.

Reply:
Interesting! Would you please file issues for both of those?

The producing a float64 result from a float32 input is concerning, and could cause considerable addition memory usage and slowdown. It'd be great to figure out what's going on there! Maybe we're mapping to the wrong nvfuser operation?